### ISSUE TITLE
Potential replacements for fasttext?

### ISSUE THREAD

**Issue by Omoeba** [languagetool-org/languagetool#10522]
The [repository](https://github.com/facebookresearch/fastText/) for `fasttext` has been archived since March 19th. As a result, it has been [deprecated](https://github.com/Homebrew/homebrew-core/commit/455493ebf4895b1ee8e053e5983df4caf9149a37) in some package managers. Are there any available alternatives now that `fasttext` is unmaintained?

**Comment by Nikola-Milovic** [comment]
Have you found any alternatives?

**Comment by Omoeba** [comment]
Nope, I just worked around it by setting preferred languages and only using those languages for automatic language detection.

**Comment by danielnaber** [comment]
It's not that fasttext is absolutely needed. It improves the language detection for short texts (<50 chars), but for longer texts its impact should be small. For cases where the language is set via a parameter, using fasttext or not shouldn't make any difference at all.

**Comment by Mahhos** [comment]
Hey guys! could you be able to find a replacement for FastText.? I am looking for a reliable alternative for text classification.

**Comment by eclecticpassions** [comment]
I'm having trouble getting the LanguageTool extension on VS Code to recognise the `properties.settings` or `languagetool.cfg` files that point to the fastTextModel and fastTextBinary. It keeps giving me this error in the VS code output: 

```
WARN  org.languagetool.language.identifier.DefaultLanguageIdentifier fastText not configured - language detection performance will be degraded. See https://dev.languagetool.org/http-server#starting-from-command-line for instructions.
```

I didn't know it's just for language detection. If I set in the extension settings `Language Tool > Language` to be `en-GB`, and not just `en` or `auto`, does that mean I don't need FastText? Thanks a lot.

**Comment by aadrian** [comment]
This fork (and improvement) of fastText  https://github.com/explosion/floret seems quite good (from the authors of spaCy)
